website/public/favicon.svg

```diff
@@ -0,0 +1,23 @@
+<svg
+  width="53"
+  height="53"
+  viewBox="0 0 53 53"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    d="m42.212 16.24-5.427-5.427-7.46 7.46a3.997 3.997 0 0 1-5.652 0l-7.46-7.46-5.428 5.427 7.46 7.46a3.997 3.997 0 0 1 0 5.653l-7.46 7.46 5.427 5.427 7.46-7.46a3.997 3.997 0 0 1 5.653 0l7.46 7.46 5.428-5.427-7.46-7.46a3.997 3.997 0 0 1 0-5.653l7.46-7.46Z"
+  />
+  <path
+    d="M26.498 41.672 21.072 47.1l5.427 5.426 5.426-5.426-5.427-5.427ZM26.499.525 21.073 5.95l5.426 5.426 5.426-5.426L26.5.525ZM47.074 21.101l-5.426 5.427 5.426 5.426 5.426-5.426-5.426-5.427ZM5.926 21.1.5 26.528l5.426 5.426 5.426-5.426-5.426-5.426Z"
+  />
+  <style>
+    path {
+      fill: #000;
+    }
```

Comment:
```suggestion
```
don't needed, and will still support 2 modes dark and white

Reply:
Even when it's loaded as favicon?

Reply:
yeah, I used the same in the-guild.dev  https://github.com/the-guild-org/website/blob/update-to-nextra-4/website/app/icon.svg?short_path=7959dfc

(Nextra 4 PR is not yet merged)

Reply:
fair dos